src/extension.ts

```diff
@@ -168,10 +218,10 @@ export function activate(context: ExtensionContext) {
 			window.showInformationMessage(`No Python3 interpreter found at "${pp}". Falling back to default virtual environment`);
 		}
 		const venvBaseDir = Uri.joinPath(context.globalStorageUri, ".env").fsPath;
-		const venvPath = Uri.joinPath(context.globalStorageUri, ".env", "bin", "python3").fsPath;
+		const venvPath = getDefaultVenvPath();
 
 		if (!fs.existsSync(venvBaseDir)) {
-			const venvProcess = cp.spawnSync("python3", ["-m", "venv", venvBaseDir]);
+			const venvProcess = cp.spawnSync("python", ["-m", "venv", venvBaseDir]);
```

Comment:
I don't think this is a good idea. Is it require to make this work on Windows?

Reply:
At first I thought so, but it turned out that I just needed to restart, nice catch.